### PR TITLE
Add summary origin

### PR DESCRIPTION
**Pull request by bfolie**

### CODE REVIEW

File: docs/specification/attributes.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Attributes are annotated with the `origin` of the data.  This field can have the
 
 - `measured`: The Value of this Attribute was directly measured.
 - `predicted`: The Value of this Attribute came from a model, such as a complex simulation, a machine learning--derived computation or rule-of-thumb estimation
+- `summary`: The Value of this Attribute is an aggregate of multiple values.
```
Comment (kroenlein) [inline docs/specification/attributes.md:37]:
```suggestion
- `summary`: The Value of this Attribute is calculated based on a set of the same property at a finer level of granularity (aggregation of data).
```
Exactly coping the text from:
https://citrineinformatics.github.io/gemd-docs/high-level-overview/#attribute-in-specs-and-runs

Possibly, we should copy the other way instead.

File: docs/specification/attributes.md [hunk 2]
```diff
@@ -50,7 +51,7 @@ Field name   | Value type | Default | Description
 `value`      | [Value](../value-types) | Req. | Any `Value` type
 `name`       | String     | Req. | The name of the attribute, which is used to identify it within a Data Object
 `notes`      | String     | None | Some free-form notes about the attribute.
-`origin`     | `measured`, `predicted`, `specified`, `computed`, `unknown` | `unknown` | The origin of the attribute
+`origin`     | `measured`, `predicted`, `summary`, `specified`, `computed`, `unknown` | `unknown` | The origin of the attribute
```
Comment (kroenlein) [inline docs/specification/attributes.md:54]: Might be worth noting that the different versions of this list have different orders.

**Review by kroenlein — COMMENTED** [review]
https://citrineinformatics.github.io/gemd-docs/faq/#where-do-i-store-statistics-about-my-measurements

I note that the directions in this block feel at-odds with the definition of Summary.  But maybe this is all scope creep relative to the initial requested fix.

**Comment by bfolie** [comment]
> I note that the directions in this block feel at-odds with the definition of Summary. But maybe this is all scope creep relative to the initial requested fix.

I agree -- Summary doesn't quite make sense as an origin. But it's implemented and exists in other places in the docs, so I figure it should also be included here.

File: docs/specification/attributes.md (hunk 2 above)
Comment (bfolie) [reply docs/specification/attributes.md:54]: It's a set, so order is 🤷

File: docs/specification/attributes.md (hunk 1 above)
Comment (bfolie) [reply docs/specification/attributes.md:37]: I agree it should be the same text in both places. I do not have a strong opinion as to the specific text.

**Review by kroenlein — APPROVED** [review]
@bfolie Resolving this outstanding issue.  Look good to you?

**Pull request merged**